### PR TITLE
Jwrapper: Fix bug with nested classes

### PR DESCRIPTION
**Pull request by Freddrickk**
Fixes the nested classes bug in jwrapper.

Splits JNI full name and java full name.
Generated comments use JNI full name.

### CODE REVIEW

File: contrib/jwrapper/src/code_generator.nit [hunk 1]
```diff
@@ -355,7 +355,7 @@ extern class {{{nit_type}}} in "Java" `{ {{{java_type.extern_equivalent}}} `}
 		end
 
 		file_out.write """
-	# Java constructor: {{{java_class}}}
+{{{t}}}# Java constructor: {{{java_class.class_type.extern_equivalent}}}
```
Comment (xymus) [inline contrib/jwrapper/src/code_generator.nit]: There's no `t` here since constructors are always within a class, and thus always indented.

**Comment by Freddrickk** [comment]
I generated queue.nit and compiled it without any warnings/errors. Any more checks to run ?

**Comment by xymus** [comment]
Could you rebase to fixup the "undo" commit into the previous one? You'll have an automatic +1 then!

**Comment by xymus** [comment]
I've tested it locally with the problematic Android package and it fixed the problem.

**Comment by Freddrickk** [comment]
Sure !

**Comment by xymus** [comment]
+1!

**Pull request merged**